packages/core/src/filters/Filter.ts

```diff
@@ -197,6 +196,8 @@ export class Filter extends Shader
      */
     public legacy: boolean;
     state: State;
+
+    private _resolution: number;
```

Comment:
As you note @mcdenhoed, `AdvancedBloomFilter` maintains a `_resolution` property already. Using the same property name and making it private will break that downstream dependency. Instead, let's make this `protected`. This will allow overriding accessors to use the same property.

```suggestion
    protected _resolution: number;
```

Reply:
Ahh, yeah, that's a good catch. Thanks!